src/main/shadow/cljs/devtools/client/hud.cljs

```diff
@@ -209,26 +209,41 @@
       (load-end)
       (dom-insert
         [:div
-         {:id hud-id
-          :style {:position "absolute"
-                  :z-index "10000"
-                  :left "0px"
-                  :bottom "0px"
-                  :right "0px"
-                  :padding "10px 10px 0 10px"
-                  :overflow "auto"
-                  :font-family "monospace"
-                  :font-size "12px"}}
-         (for [{:keys [warnings] :as src} sources-with-warnings
-               warning warnings]
-           (html-for-warning warning))])
-      )))
+         {:id    hud-id
+          :style {:position       "fixed"
+                  :z-index        "10000"
+                  :left           "0px"
+                  :bottom         "0px"
+                  :right          "0px"
+                  :display        "flex"
+                  :flex-direction "column"
+                  :font-family    "monospace"
+                  :font-size      "12px"}}
+         [:div {:style {:align-self    "flex-end"
+                        :background    "#fff"
+                        :border        "2px solid #ccc"
+                        :border-bottom "0px"
+                        :cursor        "pointer"
+                        :padding       "6px"
+                        :margin-bottom "-2px"
+                        :z-index       "1"}
```

Comment:
1?

Reply:
Yeah, just to make this effect: 
<img width="68" alt="screenshot 2018-06-04 17 08 53" src="https://user-images.githubusercontent.com/25736/40939085-073c2b84-681a-11e8-81bd-42d0e0b55a4e.png">

So the close button is over the top-border of the element below.